### PR TITLE
docs: add blog post for js-ipfs 0.53

### PR DESCRIPTION
**Pull request by achingbrain**

### CODE REVIEW

**Review by jennwrites — APPROVED** [review]
Scheduling for publication tomorrow, JAN 19. Thank you @achingbrain 👏

File: content/post/117-js-ipfs-0.53.md [hunk 1]
```diff
@@ -65,19 +65,19 @@ A few caveats:
...
 
-There's a new example called [ipfs-client-add-files](https://github.com/ipfs/js-ipfs/tree/master/examples/ipfs-client-add-files) that you can use as a sandbox to experiment with this new client.
+There’s a new example called [ipfs-client-add-files](https://github.com/ipfs/js-ipfs/tree/master/examples/ipfs-client-add-files) that you can use as a sandbox to experiment with this new client.
 
-If you fire it up, you'll see progress and file import events appearing before the upload request has completed, something that's not been possible with the `ipfs-http-client` until now.
+If you fire it up, you’ll see progress and file import events appearing before the upload request has completed, something that’s not been possible with the `ipfs-http-client` until now.
 
-What's next? We're going to extend the [gRPC][] implementation to all streaming methods and get a server into `go-IPFS` so you can use the same client with that implementation too.
+What’s next? We”re going to extend the [gRPC][] implementation to all streaming methods and get a server into `go-IPFS` so you can use the same client with that implementation too.
```
Comment (achingbrain) [inline content/post/117-js-ipfs-0.53.md:72]: @jennwrites did you mean to turn this into a double smart quote?

**Review by vasco-santos — APPROVED** [review]
This looks great!

**Review by jacobheun — APPROVED** [review]
🚢 🚀

**Review by hugomrdias — APPROVED** [review]
LGTM  🎉

File: content/post/117-js-ipfs-0.53.md (hunk 1 above)
Comment (jennwrites) [reply content/post/117-js-ipfs-0.53.md:72]: Oops no!

**Comment by jennwrites** [comment]
Thanks @achingbrain! Post is live here: https://blog.ipfs.io/2021-01-19-js-ipfs-0-50/

**Pull request merged**